### PR TITLE
Added cassandra & postgresql dashboard

### PR DESCRIPTION
**Pull request by setamyDG**

### CODE REVIEW

File: library/cassandra/config.yaml [hunk 1]
```diff
@@ -0,0 +1,11 @@
+# Name of your quickstart as shown on the website
+name: Cassandra
+description: |+
+  Describe what's the focus of your dashboards.
+  You can use multiple lines.
+  Try to be as descriptive as possible.
+
+# Optional: Authors of the quickstart
+authors:
+    - newrelic
```
Comment (kidk) [inline library/cassandra/config.yaml:10]: `New Relic`

File: library/cassandra/dashboards/cassandra.json [hunk 2]
```diff
@@ -0,0 +1,326 @@
...
+                                "accountId": 0,
+                                "query": "SELECT average(`db.totalHintsInProgress`)  * uniqueCount(entityName) AS 'In Progress' FROM CassandraSample FACET entityName TIMESERIES UNTIL 1 minute AGO"
+                            }
+                        ]
+                    }
+                }
+            ]
+        }
+    ],
+    "name": "Cassandra",
```
Comment (kidk) [inline library/cassandra/dashboards/cassandra.json:324]: Move `name` and `description` to top of JSON file.

**Review by kidk — CHANGES_REQUESTED** [review]
JSON looks good. I've added two small comments.

Also when you take the screenshot of the dashboards, could you change the time range so it shows data on the whole graph?

But besides that, everything looks great. Thanks 👍

File: library/cassandra/config.yaml (hunk 1 above)
Comment (setamyDG) [reply library/cassandra/config.yaml]: done :)

File: library/cassandra/dashboards/cassandra.json (hunk 2 above)
Comment (setamyDG) [reply library/cassandra/dashboards/cassandra.json]: done :)

**Review by kidk — CHANGES_REQUESTED** [review]
Screenshot `library/postgresql/dashboards/postgresql01.png` needs work. Just part of dashboard is fine, the menu bar is now showing multiple times.

**Review by kidk — APPROVED** [review]

**Pull request merged**